schemas/usage.json

```diff
@@ -562,37 +588,20 @@
 			"description": "The usages of various components in Elementor.",
 			"required": [
 				"posts",
+				"non-elementor-posts",
```

Comment:
Please add a "Change Log" section ( on a `md` file?) with a line "3.3.0 - added `usage/non-elementor-post` for count posts that not built with Elementor.

Reply:
Handled.